src/app.ts

```diff
@@ -33,7 +33,8 @@ mongoose.Promise = bluebird;
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useCreateIndex: true, useUnifiedTopology: true } ).then(
     () => { /** ready to use. The `mongoose.connect()` promise resolves to undefined. */ },
 ).catch(err => {
-    console.log("MongoDB connection error. Please make sure MongoDB is running. " + err);
+    console.log(`MongoDB connection error. Please make sure MongoDB is running.
+    ${err}`);
```

Comment:
```suggestion
    console.log(`MongoDB connection error. Please make sure MongoDB is running. ${err}`);
```
There is no configuration like `.prettierrc` here, IMO having the error in next line
looked like mistype.